volontulo/gulpfile.js

```diff
@@ -2,10 +2,26 @@
 
 var gulp = require('gulp');
 var gutil = require('gulp-util');
+var sass = require('gulp-sass');
 
-gulp.task('build', function() {
-    gulp.src('./frontend/**/*')
-        .pipe(gulp.dest('./static/volontulo/'));
+
+gulp.task('build_fonts', function() {
+    gulp.src('./frontend/fonts/**/*')
+        .pipe(gulp.dest('./static/volontulo/fonts/'));
+});
+
+gulp.task('build_img', function() {
+    gulp.src('./frontend/img/**/*')
+        .pipe(gulp.dest('./static/volontulo/img/'));
+});
+
+gulp.task('build_scss', function() {
+    gulp.src('./frontend/scss/**/*.scss')
+        .pipe(sass().on('error', sass.logError))
+        .pipe(gulp.dest('./static/volontulo/css/'));
+});
+
+gulp.task('build', ['build_fonts', 'build_img', 'build_scss'], function() {
```

Comment:
It looks like it's necessary https://github.com/gulpjs/gulp/blob/master/docs/API.md#gulptaskname-deps-fn